### PR TITLE
Dgo 3&a

### PR DESCRIPTION
**Pull request by dannago**
moving edits made in comments on closed PR

### CODE REVIEW

File: sp800-63-3/sec5_DIRM.md [hunk 1]
```diff
@@ -89,7 +89,7 @@ The three potential impact values are:
...
...e application must provide at least AAL2. However, the identity proofing requirements remain unclear. The entire business process of examining a résumé and ultimately hiring and onboarding a person requires a significant amount of identity proofing. The agency needs a high level of confidence that the job applicant is in fact the subject of the résumé submitted online if a decision to hire is made. Yet this level of proofing is not required to submit the résumé online. Identity proofing is not required to complete the digital portion of the transaction successfully. Identity proofing the submitter would create more risk than required in the online system as excess personal information would be collected when no such information is needed for the portion of the hiring process served by the digital job application portal and may reduce usability. Therefore, the most appropriate IAL selection would be 1. There is no need to identity proof the user to successfully complete the online transaction. This decision for the online portal itself is independent of a seemingly obvious identity proofing requirement for the entire business process, lest a job be offered to a fraudulent applicant.
```
Comment (jimfenton) [inline sp800-63-3/sec5_DIRM.md]: Why converting em dashes to two hyphens?

File: sp800-63-3/sec5_DIRM.md [hunk 2]
```diff
@@ -157,7 +157,7 @@ The statement SHALL include, at a minimum:
...
+As these guidelines are revised, CSPs SHALL consider how changes in requirements affect their user population. In some instances, the user population will be unaffected, yet in others, the CSP will require users undergo a transitional activity. For example, CSPs may request users &mdash; upon initial logon since last revision &mdash; supply additional proofing evidence  to adhere to new IAL requirements. This SHALL be a risk-based decision, made in context of the CSP, any RPs that use the CSP, mission, and the population served. The following considerations serve only as a guide to agencies when considering the impacts of requirements changes:
```
Comment (jimfenton) [inline sp800-63-3/sec5_DIRM.md]: double space after evidence

**Review by jimfenton — COMMENTED** [review]
Minor changes. Do not merge until after #1847 unless we rebase the last commit because this branch includes the 1847 changes.

**Comment by jricher** [comment]
This has been rebased and separated from #1847

File: sp800-63-3/sec5_DIRM.md (hunk 1 above)
Comment (dannago) [reply sp800-63-3/sec5_DIRM.md]: i'm having trouble getting them to render correctly - will try again

**Review by jimfenton — APPROVED** [review]
Looks good now.

**Pull request merged**